content/api/configuration.md

```diff
@@ -1,10 +1,1407 @@
 ---
 title: Configuration
 ---
-> configuration file
-> possible extensions, i. e. .babel.js
-> exporting a function and --env
-> returning a Promise
-> exporting multiple configurations
 
-> see also [[Using the Cli]]
+?> TODO: Add a reference to [[Using the Cli]]
+
+### Passing a Configuration
+
+?> TODO: exporting a function and --env
+
+?> TODO: returning a Promise
+
+?> TODO: exporting multiple configurations
+
+---
+
+### External Configurations
+
+?> TODO: possible extensions, i. e. .babel.js
+
+---
+
+Webpack is fed a configuration object. It is passed in one of two ways depending on how you are using webpack: through the Command Line (CLI) or via Node. Either method also allows an array of configuration objects, which are processed in parallel. This is more efficient than calling webpack multiple times. All the available configuration options are specified below...
```

Comment:
Making the CLI => terminal change and ellipsis fixes.

> If you pass an array to webpack does it actually execute in parallel by default?

That's what the [current docs](http://webpack.github.io/docs/configuration.html#multiple-configurations) say which is where I pulled most of this paragraph's content from. However after re-reading I decided to pull that part out as it can be discussed in **Passing a Configuration**, in fact there's already a TODO for it. I don't think passing multiple configs is something the average user is doing much, so it's probably fine to push it to a more detailed section.